src/api.cc

```diff
@@ -163,17 +163,18 @@ boost::property_tree::ptree AkliteClient::GetConfig() const {
   return pt;
 }
 
-TufTarget AkliteClient::GetCurrent() const {
-  auto current = client_->getCurrent();
+static TufTarget convertTarget(const Uptane::Target& target) {
```

Comment:
Not important. It could be TufTarget ctor that accepts `const Uptane::Target` as an input param.